### PR TITLE
Update RQ051 medication codes with new finding-missing-codes script a…

### PR DESCRIPTION
**Pull request by maiparkes58**
…nd PI review

### CODE REVIEW

File: shared/clinical-code-sets/medications/antipsychotics/2/antipsychotics.ctv3.txt [hunk 1]
```diff
@@ -0,0 +1,685 @@
...
+x02mx	Parstelin
+d84z.	Tranylcypromine 10mg tablet
+d84z.	Tranylcypromine 10mg tablet
+dw19.	Tranquilyn 5mg tablet
+dw19.	Tranquilyn 5mg tablet
+dw1A.	Tranquilyn 10mg tablet
+dw1A.	Tranquilyn 10mg tablet
+dw1B.	Tranquilyn 20mg tablet
+dw1B.	Tranquilyn 20mg tablet
+X80fL	Beta chain of CD11 a, b and c
```
Comment (rw251) [inline shared/clinical-code-sets/medications/antipsychotics/2/antipsychotics.ctv3.txt:672]: X80fL and X80gC seem unrelated to this code set - please remove all instances of them

Comment (rw251) [reply shared/clinical-code-sets/medications/antipsychotics/2/antipsychotics.ctv3.txt:672]: Also in the EMIS and the SNOMED sets remove the ones referring to "beta chain" and "antigen"

File: shared/clinical-code-sets/medications/barbituates/1/barbituates.ctv3.txt [hunk 2]
```diff
@@ -73,4 +73,12 @@ dn6x.	Methylphenobarbitone 30mg tablet
 dna2.	Mysoline 250mg/5mL oral suspension
 dna1.	Mysoline 250mg tablet
 d31f.	Soneryl 100mg tablet
+dna3.	MYSOLINE 50mg tablets
+dna3.	MYSOLINE 50mg tablets
+X78EQ	Fetal primidone syndrome
```
Comment (rw251) [inline shared/clinical-code-sets/medications/barbituates/1/barbituates.ctv3.txt:78]: Remove "fetal primidone syndrome" as this sounds like a diagnosis rather than a medication

File: shared/clinical-code-sets/medications/other-antidepressants/1/other-antidepressants.ctv3.txt [hunk 3]
```diff
@@ -62,4 +62,6 @@ x02lx	Optimax
 x02gO	Ketalar
 x02kK	Nalorex
 x040v	Cantassium vitamin B6
-x059G	Zyban
\ No newline at end of file
+x059G	Zyban
+D2102	Pyridoxine-responsive sideroblastic anaemia
```
Comment (rw251) [inline shared/clinical-code-sets/medications/other-antidepressants/1/other-antidepressants.ctv3.txt:66]: D2101 is not a medication it's a diagnosis

File: shared/clinical-code-sets/medications/other-antidepressants/1/other-antidepressants.readv2.txt [hunk 4]
```diff
@@ -49,4 +49,5 @@ du6z.11	AMFEBUTAMONE HYDROCHLORIDE 150mg m/r tablets
 du6z.00	BUPROPION HYDROCHLORIDE 150mg m/r tablets
 ld1x.00	CETYLPYRIDINIUM CL+ASCORBIC ACID 1.4mg/125mg lozenges
 ld1m.00	CLIOQUINOL+ASCORBIC ACID 35mg/6mg lozenges
-D210200	Pyridoxine-responsive sideroblastic anaemia
\ No newline at end of file
+D210200	Pyridoxine-responsive sideroblastic anaemia
```
Comment (rw251) [inline shared/clinical-code-sets/medications/other-antidepressants/1/other-antidepressants.readv2.txt:52]: Also remove this anaemia code.

File: shared/clinical-code-sets/medications/other-antidepressants/1/other-antidepressants.snomed.txt [hunk 5]
```diff
@@ -90,4 +90,20 @@
 725893006	N-methyl-L-tryptophan
 741014002	Tryptophan hydroxylase inhibitor
 443311000124103	Lysine and tryptophan-free infant formula
-444231000124103	Lysine and tryptophan-free formula
\ No newline at end of file
+444231000124103	Lysine and tryptophan-free formula
+11143411000001100	Opizone 50mg tablets (Britannia Pharmaceuticals Ltd) (product)
+15457711000001104	Valdoxan 25mg tablets (Servier Laboratories Ltd) (product)
+15499111000001100	Agomelatine 25mg tablets (product)
+191260004	Pyridoxine-responsive sideroblastic anemia (disorder)
```
Comment (rw251) [inline shared/clinical-code-sets/medications/other-antidepressants/1/other-antidepressants.snomed.txt:97]: And remove this anaemia as well

File: shared/clinical-code-sets/medications/other-anxiolytics-and-hypnotics/1/other-anxiolytics-and-hypnotics.readv2.txt [hunk 6]
```diff
@@ -1,17 +1,69 @@
...
+do3..00	CLOMETHIAZOLE EDISYLATE [CENTRAL NERVOUS SYSTEM USE]
+do3z.00	CLOMETHIAZOLE EDISYLATE 8mg/mL intravenous infusion
+dibT.00	PARACETAMOL+DIPHENHYDRAMINE HYDROCHLORIDE 120mg/12.5mg/5mL sugar free oral solution
+dibO.00	PARACETAMOL+DIPHENHYDRAMINE HYDROCHLORIDE 500mg/25mg tablets
+d3...00	BARBITURATES/METHYPRYLONE
+d32..00	*METHYPRYLONE
+dC1..00	SODIUM OXYBATE
+dC1z.00	SODIUM OXYBATE 500mg/mL oral solution
+m347.00	DIPHENHYDRAMINE HYDROCHLORIDE 2% cream
+9Ngt.00	On melatonin for sleep disorder
```
Comment (rw251) [inline shared/clinical-code-sets/medications/other-anxiolytics-and-hypnotics/1/other-anxiolytics-and-hypnotics.readv2.txt:68]: This isn't a medication code - but it does indicate that the person is taking the drug. The problem is that this code would not appear in the GP_Medications table (would be in GP_Events), so you'd need to remember to search both tables when using this code set. In the past for e.g. flu vaccine codes where there are medication codes (the actual vaccine), and procedure codes (the vaccination event) which are both used, I have created a "flu-vaccine" code set under medications, and a "flu-vaccination" code set under procedures. You can then search GP_Medications for the first, and GP_Events for the second. Having said all that, in this instance I suspect you can remove the 9Ngt code from this list as there will be no-one on the drug who has this code, but doesn't have a medication code.

File: shared/clinical-code-sets/medications/serotonin-modulator-stimulator/1/README.md [hunk 7]
```diff
@@ -6,10 +6,10 @@ This code set was created from getset.ga, using the following list from the PI o
 
 ## Prevalence log
 
-By examining the prevalence of codes (number of patients with the code in their record) broken down by clinical system, we can attempt to validate the clinical code sets and the reporting of the conditions. Here is a log for this code set. The prevalence range `... - ...%` suggests that this code set is well defined.
+By examining the prevalence of codes (number of patients with the code in their record) broken down by clinical system, we can attempt to validate the clinical code sets and the reporting of the conditions. Here is a log for this code set. The prevalence range `0.001 - 0.003%` suggests that this code set is well defined.
```
Comment (rw251) [inline shared/clinical-code-sets/medications/serotonin-modulator-stimulator/1/README.md:9]: The narrow range might suggest it is well defined, but with only 40 patients in the whole of GM with this in their record I'd say it's either missing codes, or is something that isn't prescribed in primary care. You get some meds that only ever get prescribed in hospital.

**Review by rw251 — CHANGES_REQUESTED** [review]
In addition to the various code comments, I've noticed that my script to find other codes, didn't work as expected for code sets with more than 1 version. For this pr it's just antipsychotics and benzodiazepines. I fixed the script, and ran it for benzos, and have added the extra EMIS and SNOMED codes to this branch. However I haven't updated the prevalance for benzos, and I haven't done anything for antipsychotics - so that needs the find missing codes logic running again.

File: shared/clinical-code-sets/medications/serotonin-modulator-stimulator/1/README.md (hunk 7 above)
Comment (maiparkes58) [reply shared/clinical-code-sets/medications/serotonin-modulator-stimulator/1/README.md:9]: I see. The finding-missing-code script was used to look for potential missing codes. What should we do with this medication group then? Do we need to note with the PI?

**Comment by maiparkes58** [comment]
Thanks for reviewing. All updates are made based on your comments.

File: shared/clinical-code-sets/medications/serotonin-modulator-stimulator/1/README.md (hunk 7 above)
Comment (rw251) [reply shared/clinical-code-sets/medications/serotonin-modulator-stimulator/1/README.md:9]: Yes - let the PI know. Show them the codes you have, and ask if they would expect more than 40 people in GM to have ever had this prescribed by a GP.

**Review by rw251 — APPROVED** [review]
Thanks for the changes. All looks good now.

File: shared/clinical-code-sets/medications/serotonin-modulator-stimulator/1/README.md (hunk 7 above)
Comment (maiparkes58) [reply shared/clinical-code-sets/medications/serotonin-modulator-stimulator/1/README.md:9]: OK I just email the PI.
